rust/private/rustc.bzl

```diff
@@ -791,6 +791,9 @@ def construct_arguments(
         rustc_flags.add("--extern")
         rustc_flags.add("proc_macro")
 
+    if ctx.configuration.coverage_enabled:
+        rustc_flags.add("-Cinstrument-coverage")
```

Comment:
This was only added pretty recently, maybe even 1.60 even, not sure if we should try to gate something about this? Otherwise it was unstable / nightly only